Makefile

```diff
@@ -1,13 +1,7 @@
 include scripts/common.Makefile
 
-
-#Variable need to be set in .env file according to swarm deploy at hand
-# Internal VARIABLES ------------------------------------------------
-.env: .env-config ## creates .env file from defaults in .env-config
-	$(if $(wildcard $@), \
-	@echo "WARNING #####  $< is newer than $@ ####"; diff -uN $@ $<; false;,\
-	@echo "WARNING ##### $@ does not exist, cloning $< as $@ ############"; cp $< $@ && \
-	sed -i '/\$$/d' $@ && export $$(grep -v '^#' $@ | xargs) && envsubst < $< > $@)
+.env:
+	touch .env
```

Comment:
not sure I understand that one. you need to have one even if it is empty?

Reply:
yes